element/index.js

```diff
@@ -65,3 +66,20 @@ export function renderToString( element ) {
 
 	return renderToStaticMarkup( element );
 }
+
+export function concatValues( value1, value2 ) {
```

Comment:
>`concatChildren` ?

I like this, as it seems like if this were a function native to React, it'd be called `React.Children.concat`